File: SDKs/Aspose.Slides-Cloud-SDK-for-.NET/src/Com/Aspose/Slides/Model/TextItem.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Slides.Model {
  public class TextItem {
    public ResourceUri Uri { get; set; }

    public string Text { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class TextItem {\n");
      sb.Append("  Uri: ").Append(Uri).Append("\n");
      sb.Append("  Text: ").Append(Text).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Slides-Cloud-SDK-for-Android/Aspose-Slides-Cloud-SDK-for-Android/src/main/java/com/aspose/slides/model/TextItem.java
package com.aspose.slides.model;

import com.aspose.slides.model.ResourceUri;
public class TextItem {
  private ResourceUri Uri = null;
  private String Text = null;
  /**
	 * getUri
	 * Gets ResourceUri
	 * @return Uri
	 */
  public ResourceUri getUri() {
    return Uri;
  }

	/**
	 * setUri
	 * Sets ResourceUri
	 * @param Uri ResourceUri
	 */
  public void setUri(ResourceUri Uri) {
    this.Uri = Uri;
  }

  /**
	 * getText
	 * Gets String
	 * @return Text
	 */
  public String getText() {
    return Text;
  }

	/**
	 * setText
	 * Sets String
	 * @param Text String
	 */
  public void setText(String Text) {
    this.Text = Text;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class TextItem {\n");
    sb.append("  Uri: ").append(Uri).append("\n");
    sb.append("  Text: ").append(Text).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

